reviews/NFTTrader/testing.md

```diff
@@ -0,0 +1,48 @@
+# Testing
+
+
+## Summary
+
+- I could not get these tests to run properly - I'm not sure if there was additional config or project files that were removed for the audit. It would have helped if these had been included.
+
+### Recommendations
+- Looks like the test file starts out with a bunch of helper functions. It might be nice to label this or split them out.
+- In general the test code is repetitive and could definitely be refactored.
```

Comment:
yes , in other sense , following  the standards  example as shown [here](https://github.com/adrianmcli/ganache-jest-example/blob/master/test.js).
- defining the constants of mocking the  users.
- defining the conditions for each of the function 
- using assertions